tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -220,14 +217,20 @@ def test_container_both_kinds_same_sub(eof_test: EOFTestFiller):
                 revert_sub_container,
             ],
         ),
-        expect_exception=EOFException.UNDEFINED_EXCEPTION,
+        expect_exception=EOFException.INCOMPATIBLE_CONTAINER_KIND,
     )
 
 
-def test_container_both_kinds_different_sub(eof_test: EOFTestFiller):
-    """Test multiple kinds of subcontainer at the same level"""
+@pytest.mark.parametrize("kind", [ContainerKind.RUNTIME, ContainerKind.INITCODE])
```

Comment:
I think the validation should fail for both kinds as described in the comment, however evmone thinks the "initcode" validation is fine.